optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/lang/tokens/OptionalEmployee.java

```diff
@@ -0,0 +1,32 @@
+package org.optaplanner.openshift.employeerostering.shared.lang.tokens;
+
+import javax.annotation.Nullable;
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+
+import org.optaplanner.openshift.employeerostering.shared.common.AbstractPersistable;
+import org.optaplanner.openshift.employeerostering.shared.employee.Employee;
+
+@Entity
+public class OptionalEmployee extends AbstractPersistable {
```

Comment:
Why do we need this class? Can we get rid of it by just allow null's of Employee?

Reply:
It needed as `Employee` are used in a List, and Hibernate doesn't behave well with nulls in a list to say the least... (and I cannot use the `Optional<Employee>` class since Hibernate isn't aware of it). (See line 92 of `ShiftInfo`)